### DISCUSSION TITLE
Ambiguous annotation results

### DISCUSSION THREAD

**Discussion by cheanney** [KarchinLab/open-cravat#238] [Q&A]
First, I would like to say thank you for this wonderful too. This tool has significantly improved our workflow.

However, we've encountered some issues with ambiguous annotation results. Specifically, we're analyzing the following genetic variant:
chr7 149528263 + G A
This variant is based on the HG19 reference.

Using the openCravet platform, this variant is identified as intergenic. Yet, it's associated with COSMIC annotations as follows:
COSMIC_ID: COSV65128555
COSMIC_TISSUES: [["NS", 1], ["large intestine", 1]]
COSMIC_VARIANT_COUNT: 2
COSMIC_TRANSCRIPT: ENST00000378016.4
COSMIC_PROTEIN_CHANGE: R5063Q
COSMIC_MATCH: Exact Match

We're seeking clarity on these conflicting annotations.

Thanks, 

Anney

<img width="1413" alt="Screenshot 2024-03-20 at 2 05 37 PM" src="https://github.com/KarchinLab/open-cravat/assets/8795101/dbc67e09-2ee9-44ad-8618-81e12a78bc5a">

**Comment by kmoad** [comment]
This variant maps to a transcribed pseudogene, SSPOP/SSPO. The exact variant has annotation in COSMIC https://cancer.sanger.ac.uk/cosmic/mutation/overview?id=116634497. We don't map pseudogene variants even when they are transcribed, we instead report them as intergenic.

We do offer a `pseudogene` annotator, which annotates this variant with the same transcript as COSMIC.
